### PR TITLE
Feat/verification webhooks

### PR DESCRIPTION
**Pull request by Dovchik**
Add hooks classes + minor namespaces regrouping

### CODE REVIEW

File: src/Sinch/Verification/Common/VerificationMethod.cs [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+using Sinch.Core;
 
-namespace Sinch.Verification
+namespace Sinch.Verification.Common
 {
     /// <summary>
     ///     The method of the verification request.
     /// </summary>
     /// <param name="Value"></param>
+    [JsonConverter(typeof(EnumRecordJsonConverter<VerificationMethod>))]
```
Comment (JPPortier) [inline src/Sinch/Verification/Common/VerificationMethod.cs:10]: https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-callbacks/ is defining 4th value for response: `seamless`

File: src/Sinch/Verification/Hooks/VerificationResultEvent.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        ///     Gets or sets the type of the event.
+        /// </summary>
+        [JsonPropertyName("event")]
+        public string Event { get; set; }
+
+        /// <summary>
+        ///     Gets or sets the verification method. Must be one of the following: "sms", "flashCall", "callout".
+        /// </summary>
+        [JsonPropertyName("method")]
+        public VerificationMethod Method { get; set; }
```
Comment (JPPortier) [inline src/Sinch/Verification/Hooks/VerificationResultEvent.cs:32]: see comment about VerificationMethod and documentation
Response can contains `seamless` value

Documentaiton inconsistency ?

Comment (Dovchik) [reply src/Sinch/Verification/Hooks/VerificationResultEvent.cs:32]: [VerificationResultEvent has seamless](https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-callbacks/#tag/Verification-callbacks/paths/VerificationResultEvent/post)
Where [VerificationRequest event](https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-callbacks/#tag/Verification-callbacks/paths/VerificationRequestEvent/post!path=method&t=request) doesn't

Comment (JPPortier) [reply src/Sinch/Verification/Hooks/VerificationResultEvent.cs:32]: ¯\_(ツ)_/¯

**Review by JPPortier — APPROVED** [review]

**Pull request merged**